src/utils.js

```diff
@@ -127,25 +127,62 @@ exports.hideAbsolutePathsInObject = (object) => {
   }
 
   if (_.isString(object)) {
-    const words = object.split(' ').map(word => (path.isAbsolute(word) ? path.basename(word) : word))
-    return words.join(' ')
+    const words = object.split(' ')
+    .map(word => (path.isAbsolute(word) ? path.basename(word) : word))
+    .join(' ')
+    const anonymizedWords = exports.anonymizePaths(words)
```

Comment:
Line 131, every absolute path is transformed to its basename: `/1/2/3` -> `3`
Why running `anonymizePaths` on these?

Reply:
Because it doesn't cover the case of a string not being *just* a path (which is what `path.isAbsolute` checks) but containing a path. It could still use some improvements though